packages/front-end/pages/datasources/[did].tsx

```diff
@@ -260,123 +207,36 @@ mixpanel.init('YOUR PROJECT TOKEN', {
               </p>
 
               <div className="card py-3 px-3 mb-4">
-                {/* TODO: design changes for Identity Joins nested */}
-                {/* region Identifier Types */}
```

Comment:
Each section lives in its own component. The region tags for code organization are no longer necessary and looked redundant now that the whole page is refactored.